web/themes/custom/hoeringsportal/hoeringsportal.theme

```diff
@@ -329,6 +329,15 @@ function hoeringsportal_preprocess_ds_1col(&$variables) {
 function hoeringsportal_preprocess_ds_2col_stacked(&$variables) {
   $node_type = isset($variables['content']['#bundle']) ? $variables['content']['#bundle'] : '';
   // Check content type.
+  $itk_config = \Drupal::getContainer()->get('itk_admin.itk_config');
+  $node_newsletter = $itk_config->get('node_newsletter');
+  $variables['node_newsletter'] = $node_newsletter ?? NULL;
+  $variables['show_newsletter_signup'] = FALSE;
+  if ($node_newsletter === $variables['content']['#node']->id()) {
+    $variables['show_newsletter_signup'] = TRUE;
+    $variables['newsletter_iframe_source'] = $itk_config->get('newsletter_iframe_source');
+    $variables['newsletter_iframe_height'] = $itk_config->get('newsletter_iframe_height');
+  }
```

Comment:
This should be moved above the `// Check content type.` line.

And a new comment telling that we're showing the newsletter sign-up one the selected `node_newsletter` page may be useful.

Reply:
Added comment and moved code